dotCMS/src/main/java/com/dotcms/rest/api/v1/page/PageResource.java

```diff
@@ -47,39 +61,71 @@ protected PageResource(HTMLPageAssetAPI pageApi, HostAPI hostApi, WebResource we
      * <p/>
      * Usage: /page/{hostOrFolderIdentifier}
      */
-    @GET
-    @JSONP
-    @Path("/page/{pageId}")
     @NoCache
-    @Produces({MediaType.APPLICATION_JSON, "application/javascript"})
-    public Map<String, String> list(@Context HttpServletRequest request, @PathParam("pageId") String pageId) {
-
-        Map<String, String> hash = Maps.newHashMap();
-
-
-        return hash;
-    }
-
-    /**
-     * <p>Returns a JSON representation of the Rule with the given ruleId
-     * <p/>
-     * Usage: GET api/rules-engine/sites/{siteId}/rules/{ruleId}
-     */
     @GET
-    @JSONP
-    @Path("/rules/{ruleId}")
-    @NoCache
     @Produces({MediaType.APPLICATION_JSON, "application/javascript"})
-    public String self(@Context HttpServletRequest request, @PathParam("siteId") String siteId, @PathParam("ruleId") String ruleId) {
-        Map<String, String> hash = Maps.newHashMap();
-
-
-        return "asd";
+    @Path("/json/{uri: .*}")
+    public Response loadJson(@Context HttpServletRequest request, @Context HttpServletResponse
+            response, @PathParam("uri") String uri) {
+        // Force authentication
+        final InitDataObject auth = webResource.init(false, request, false);
```

Comment:
question: does support anonymous users? if do not the webResource.init should be
```
 webResource.init(false, request, true);
```